src/pages/components/toggle.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { graphql } from 'gatsby';
+import { NamespacesConsumer } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import { Toggle } from 'components/ExampleComponents';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import Section from 'components/Section';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+
+const Page: React.FC = (): React.ReactElement => {
+  const [isChecked, setChecked] = useState(false);
+
+  const getAriaLabelText = (t: Function): string =>
+    t(`toggle.state.${isChecked ? 'on' : 'off'}`, {
+      name: t('toggle.label'),
+    });
+
+  const handleClick = (newState: boolean): void => {
```

Comment:
In general I favor the "handle*element*Click" as in "handleToggleClick" naming convention, but in this case it doesn't really make a difference.

Reply:
Agreed, even when in encapsulated scope with no issues with potential naming conflicts. Using more explicit naming improves code readability and reduces the need for commenting the implementation.